docs/installation.md

```diff
@@ -0,0 +1,251 @@
+```
+
+You do not need to rebuild the image for configuration changes, it will 
+suffice to edit `config.json` and restart the container.
+
+
+# Linux / MacOS
+## 1. Requirements
+Click each one for install guide:
+- [Python 3.6.x](http://docs.python-guide.org/en/latest/starting/installation/), 
+note the bot was not tested on Python >= 3.7.x
+- [pip](https://pip.pypa.io/en/stable/installing/)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [virtualenv](https://virtualenv.pypa.io/en/stable/installation/) (Recommended)
+- [TA-Lib](https://mrjbq7.github.io/ta-lib/install.html)
+
+## 2. First install required packages
+This bot require Python 3.6 and TA-LIB
+
+### 2.1 Linux - Ubuntu 16.04
+
+**2.1.1. Install Python 3.6, Git, and wget**
+```bash
+sudo add-apt-repository ppa:jonathonf/python-3.6
+sudo apt-get update
+sudo apt-get install python3.6 python3.6-venv build-essential autoconf libtool pkg-config make wget git
+```
+
+**2.1.2. Install TA-LIB**  
+Official webpage: https://mrjbq7.github.io/ta-lib/install.html
+```
+wget http://prdownloads.sourceforge.net/ta-lib/ta-lib-0.4.0-src.tar.gz
+tar xvzf ta-lib-0.4.0-src.tar.gz
+cd ta-lib
+./configure --prefix=/usr
+make
+make install
+cd ..
+rm -rf ./ta-lib*
+```
+
+**2.1.3. [Optional] Install MongoDB**  
+Install MongoDB if you plan to optimize your strategy with Hyperopt.
+
+```bash
+sudo apt-get install mongodb-org
+```
+Complete tutorial on [Digital Ocean: How to Install MongoDB on Ubuntu 16.04](https://www.digitalocean.com/community/tutorials/how-to-install-mongodb-on-ubuntu-16-04)
+
+### 2.2. Linux - Other distro
+If you are on a different Linux OS you maybe have to adapt things like:
+
+- package manager (for example yum instead of apt-get)
+- package names
+
+### 2.3. MacOS installation
+
+**2.3.1. Install git and wget**
+```bash
+brew install wget git
+```
+
+**2.3.2. Install Python 3.6**  
+- [Python 3.6 MacOS offical page](https://www.python.org/downloads/mac-osx/)
+
+
+**2.3.3. [Optional] Install MongoDB**  
+Install MongoDB if you plan to optimize your strategy with Hyperopt.
+```bash
+curl -O https://fastdl.mongodb.org/osx/mongodb-osx-ssl-x86_64-3.4.10.tgz
+tar -zxvf mongodb-osx-ssl-x86_64-3.4.10.tgz
+mkdir -p <path_freqtrade>/env/mongodb
+cp -R -n mongodb-osx-x86_64-3.4.10/ <path_freqtrade>/env/mongodb
+export PATH=<path_freqtrade>/env/mongodb/bin:$PATH
+```
+
+## 3. Clone the repo
+The following steps are made for Linux/mac environment
+1. Clone the git `git clone git@github.com:gcarq/freqtrade.git`
+2. (Optional) Checkout the develop branch `git checkout develop` 
+
+## 4. Prepare the bot
+```bash
+cd freqtrade
+cp config.json.example config.json
+```
+To edit the config please refer to [Bot Configuration](https://github.com/gcarq/freqtrade/blob/develop/docs/configuration.md) 
+
+## 5. Setup your virtual env
+```bash
+python3.6 -m venv .env
+source .env/bin/activate
+pip3.6 install -r requirements.txt
+pip3.6 install -e .
+pip3.6 install hyperopt
```

Comment:
Not needed, as hyperopt is in `requirements.txt`

Reply:
Removed